.travis.yml

```diff
@@ -13,8 +11,17 @@ install:
   - pip install codecov
 
 script:
-  - flake8 docker_ci_deploy
+  - flake8 .
   - py.test --cov docker_ci_deploy docker_ci_deploy
```

Comment:
Maybe run `flake8` after `py.test`? We don't want a style violation to mask failing tests.